apps/api/src/routes/users.ts

```diff
@@ -0,0 +1,9 @@
+import { Hono } from "hono"
+import { PrivateContextVariables } from "@alittlebyte/api/utils/types"
+import { creditCardRouter } from "@alittlebyte/api/routes/users/creditCards"
+
+export const usersRouter = () =>
+	new Hono<{ Variables: PrivateContextVariables }>().route(
+		"/:userId/credit-cards",
+		creditCardRouter(),
```

Comment:
Add a middleware to check if the connected user id is equal to the userId param 
```suggestion
                isLoginUser,
		creditCardRouter(),
```